packages/engine.test/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@c11/engine.test",
+  "version": "4.0.0-alpha.5",
+  "description": "A test lib for the engine",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "license": "MIT",
+  "author": {
+    "name": "Alexandru Simandi",
+    "email": "simandialexander@gmail.com"
+  },
+  "contributors": [],
+  "scripts": {
+    "build": "tsc",
+    "test": "jest --clear-cache && jest --config ../../jest.config.js --runTestsByPath ./specs/*",
+    "test:coverage": "jest --config ../../jest.config.js --collectCoverage --runTestsByPath   ./src/**"
+  },
+  "peerDependencies": {
+    "react": "^16"
+  },
+  "dependencies": {
+    "@c11/engine.react": "^4.0.0-alpha.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
+    "@babel/preset-typescript": "^7.10.4",
+    "@testing-library/react": "^11.0.4",
```

Comment:
I don't think this dependency is used. Should be removed.

Reply:
without having that much experience with typescript i don't know where to stop with cutting the libs without manually bruteforcing to find all unused